gcloud/monitoring/_dataframe.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Time series as `pandas`_ dataframes.
+
+.. _pandas: http://pandas.pydata.org/pandas-docs/stable/
+"""
+
+import itertools
+
+TOP_RESOURCE_LABELS = [
+    'project_id',
+    'aws_account',
+    'location',
+    'region',
+    'zone',
+]
+
+
+def _build_dataframe(time_series_iterable,
+                     label=None, labels=None):  # pragma: NO COVER
+    """Build a ``pandas`` dataframe out of time series.
+
+    :type time_series_iterable:
+        iterable over :class:`~gcloud.monitoring.timeseries.TimeSeries`
+    :param time_series_iterable:
+        An iterable (e.g., a query object) yielding time series.
+
+    :type label: string or None
+    :param label:
+        The label name to use for the dataframe header. This can be the name
+        of a resource label or metric label (e.g., ``"instance_name"``), or
+        the string ``"resource_type"``.
```

Comment:
I don't really care but our "style" would be instead

```python
    :type label: string or None
    :param label: The label name to use for the dataframe header. This can
                  be the name of a resource label or metric label (e.g.,
                  ``"instance_name"``), or the string ``"resource_type"``.
```